{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "从Adidas中国官网爬取所有线下店的位置、类型等数据。[其官网](https://www.adidas.com.cn/location/storefinder?locale=zh_CN),它是用json穿搜索数据的，所有比较容易的，不过需要主动设置Cookie，否则是会被forbidden的。"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "import time\n",
    "import requests\n",
    "import traceback\n",
    "from lxml import etree"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 爬取"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "province={130000: \"河北省\",\n",
    "        540000: \"西藏\",\n",
    "        330000: \"浙江省\",\n",
    "        620000: \"甘肃省\",\n",
    "        360000: \"江西省\",\n",
    "        210000: \"辽宁省\",\n",
    "        500000: \"重庆\",\n",
    "        370000: \"山东省\",\n",
    "        460000: \"海南省\",\n",
    "        530000: \"云南省\",\n",
    "        520000: \"贵州省\",\n",
    "        340000: \"安徽省\",\n",
    "        220000: \"吉林省\",\n",
    "        630000: \"青海省\",\n",
    "        510000: \"四川省\",\n",
    "        150000: \"内蒙古\",\n",
    "        420000: \"湖北省\",\n",
    "        640000: \"宁夏\",\n",
    "        450000: \"广西\",\n",
    "        650000: \"新疆\",\n",
    "        310000: \"上海\",\n",
    "        230000: \"黑龙江省\",\n",
    "        410000: \"河南省\",\n",
    "        140000: \"山西省\",\n",
    "        110000: \"北京\",\n",
    "        440000: \"广东省\",\n",
    "        320000: \"江苏省\",\n",
    "        120000: \"天津\",\n",
    "        430000: \"湖南省\",\n",
    "        610000: \"陕西省\",\n",
    "        350000: \"福建省\"} \n",
    "#这部分没找到json，直接从https://img.adidas.com.cn/??libs/spice/js/jquery.spice.tools.js里抄"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "def getOneUrl(url,w=0): #获取一个页面\n",
    "    try:\n",
    "        headers = {'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36', \n",
    "                   'Cookie': 'JSESSIONID=A4916E70EDC6F89BE6D9925544E9B08C-n51; _ga=GA1.3.102630160.1556376306; _gid=GA1.3.353691624.1556376306; XSRF-TOKEN=6af9bf7f-9fcb-4e17-a2b0-5ea533870bd5; adidas_recommend_cookie_key=%u62A2YEEZY%2CClima%2CNite%20JOGGER%2CGame%20of%20Thrones%2Cultraboost%2Cyung%2096%2Csuperstar%2Cstan%20smith%2C%u4E09%u53F6%u8349%2CNMD; s_c_c_n=0; _Jo0OQK=74552BCA648073AE899B28A0D522495E51EAB059F5523F1D374782338C04ED87FAE8FB1E8F487E7835F55EA6C8AE8086E88B7899FF054733A798F56662BF2EEBDE7CD70587E0A40CF3EE1B24C75C2FEBCEFE1B24C75C2FEBCEF61332EE600EB2D15FA5A47021B17D55BGJ1Z1LA==; pt_ref_7f28852c=; pt_s_7f28852c=vt=1556384159877&cad=; pt_7f28852c=uid=2sXBoRWY1OVmX3urOjQavQ&nid=0&vid=RbuqYYxba1lXaOrzBZtR4Q&vn=3&pvn=1&sact=1556385728025&to_flag=1&pl=udUDxHiiaEhlGzijk/Jvrw*pt*1556385669042'}\n",
    "        r=requests.get(url,headers=headers,timeout=30)\n",
    "        #r.encoding=r.apparent_encoding\n",
    "        return r.text\n",
    "    except:\n",
    "        \n",
    "        try:\n",
    "            print(str(r),r.text)\n",
    "        except:\n",
    "            print('ccc',w,url)\n",
    "        #print(str(r),r.text)\n",
    "        traceback.print_exc()\n",
    "        if w>3:\n",
    "            return None\n",
    "        else:\n",
    "            time.sleep(w)\n",
    "            w+=1\n",
    "            txt=getOneUrl(url,w)\n",
    "            if txt==None:\n",
    "                print(i,url) #continue\n",
    "            else:\n",
    "                return txt\n",
    "        return None\n",
    "\n",
    "def getStoreTypes(city):#input: a city output alist\n",
    "    stlst=[]\n",
    "    url='https://www.adidas.com.cn/location/storetype.json?city={0}&_=1556377484672'.format(c)\n",
    "    rtxt=getOneUrl(url)\n",
    "    if rtxt==None:\n",
    "        print('rtxt==None',city,url)\n",
    "        rtxt='[]'\n",
    "    sts=json.loads(rtxt)\n",
    "    #like:[{\"id\":0,\"storeType\":\"运动休闲概念店\",\"storeTypeEN\":\"Sportswear Collective (SWC)\"},{\"id\":0,\"storeType\":\"运动表现店\",\"storeTypeEN\":\"Sport Performance\"},{\"id\":0,\"storeType\":\"折扣店\",\"storeTypeEN\":\"Outlet\"}]\n",
    "    for sd in sts: #sotretypes\n",
    "        stlst.append(sd['storeType'])\n",
    "    return stlst\n",
    "  \n",
    "def getSTotal(pv,c,store):\n",
    "    url='https://www.adidas.com.cn/location/storelist.json?startPage=1&size=12&province={0}&city={1}&storeName=&types%5B%5D={2}&_=1556376900755'.format(pv,c,store)\n",
    "    rtxt=getOneUrl(url)\n",
    "    if rtxt==None:\n",
    "        print('getSTotal==None',pv,c,url)\n",
    "        stt=''\n",
    "    stt=json.loads(rtxt)\n",
    "    return [stt['storeTotal'],stt['storeMapResult']]\n",
    "\n",
    "#https://www.adidas.com.cn/location/storelist.json?startPage=1&size=12&province=&city=%E5%8C%97%E4%BA%AC%E5%B8%82&storeName=&_=1556390707883"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "res={}\n",
    "klst=[]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "河北 1 西藏 2 浙江 3 甘肃 4 江西 5 辽宁 6 ccc https://www.adidas.com.cn/location/storetype.json?city=新民市&_=1556377484672\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad166f198>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E6%96%B0%E6%B0%91%E5%B8%82&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad166f198>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E6%96%B0%E6%B0%91%E5%B8%82&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad166f198>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ccc https://www.adidas.com.cn/location/storetype.json?city=新民市&_=1556377484672\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad166f198>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E6%96%B0%E6%B0%91%E5%B8%82&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad166f198>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E6%96%B0%E6%B0%91%E5%B8%82&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad166f198>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad16a23c8>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E6%96%B0%E6%B0%91%E5%B8%82&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad16a23c8>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E6%96%B0%E6%B0%91%E5%B8%82&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad16a23c8>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "重庆 7 山东 8 海南 9 云南 10 贵州 11 安徽 12 吉林 13 青海 14 四川 15 内蒙古 16 湖北 17 宁夏 18 广西 19 新疆 20 上海 21 黑龙江 22 河南 23 山西 24 北京 25 广东 26 ccc https://www.adidas.com.cn/location/storetype.json?city=阳东县&_=1556377484672\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0af5ac8>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E9%98%B3%E4%B8%9C%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0af5ac8>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E9%98%B3%E4%B8%9C%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0af5ac8>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ccc https://www.adidas.com.cn/location/storetype.json?city=阳东县&_=1556377484672\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0af5ac8>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E9%98%B3%E4%B8%9C%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0af5ac8>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E9%98%B3%E4%B8%9C%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0af5ac8>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0b38518>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E9%98%B3%E4%B8%9C%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0b38518>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E9%98%B3%E4%B8%9C%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0b38518>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "江苏 27 天津 28 湖南 29 陕西 30 福建 31 ccc https://www.adidas.com.cn/location/storetype.json?city=连江县&_=1556377484672\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905438>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E8%BF%9E%E6%B1%9F%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905438>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E8%BF%9E%E6%B1%9F%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905438>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ccc https://www.adidas.com.cn/location/storetype.json?city=连江县&_=1556377484672\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905438>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E8%BF%9E%E6%B1%9F%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905438>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E8%BF%9E%E6%B1%9F%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905438>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0a1a550>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E8%BF%9E%E6%B1%9F%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0a1a550>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E8%BF%9E%E6%B1%9F%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0a1a550>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ccc https://www.adidas.com.cn/location/storetype.json?city=闽侯县&_=1556377484672\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905c88>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E9%97%BD%E4%BE%AF%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905c88>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E9%97%BD%E4%BE%AF%E5%8E%BF&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905c88>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ccc https://www.adidas.com.cn/location/storetype.json?city=南安市&_=1556377484672\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922828>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E5%8D%97%E5%AE%89%E5%B8%82&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922828>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storetype.json?city=%E5%8D%97%E5%AE%89%E5%B8%82&_=1556377484672 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922828>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ccc https://www.adidas.com.cn/location/storelist.json?startPage=1&size=12&province=福建&city=泉州市&storeName=&types%5B%5D=童装店&_=1556376900755\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ccc https://www.adidas.com.cn/location/storelist.json?startPage=1&size=12&province=福建&city=泉州市&storeName=&types%5B%5D=童装店&_=1556376900755\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad092a710>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad092a710>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad092a710>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ccc https://www.adidas.com.cn/location/storelist.json?startPage=1&size=12&province=福建&city=泉州市&storeName=&types%5B%5D=童装店&_=1556376900755\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0922550>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad092a710>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad092a710>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad092a710>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 141, in _new_conn\n",
      "    (self.host, self.port), self.timeout, **extra_kw)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/connection.py\", line 60, in create_connection\n",
      "    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/socket.py\", line 745, in getaddrinfo\n",
      "    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):\n",
      "socket.gaierror: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 601, in urlopen\n",
      "    chunked=chunked)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 346, in _make_request\n",
      "    self._validate_conn(conn)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 850, in _validate_conn\n",
      "    conn.connect()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 284, in connect\n",
      "    conn = self._new_conn()\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connection.py\", line 150, in _new_conn\n",
      "    self, \"Failed to establish a new connection: %s\" % e)\n",
      "urllib3.exceptions.NewConnectionError: <urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905908>: Failed to establish a new connection: [Errno -2] Name or service not known\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 440, in send\n",
      "    timeout=timeout\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py\", line 639, in urlopen\n",
      "    _stacktrace=sys.exc_info()[2])\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/urllib3/util/retry.py\", line 388, in increment\n",
      "    raise MaxRetryError(_pool, url, error or ResponseError(cause))\n",
      "urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905908>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"<ipython-input-5-c2fdd8c7672b>\", line 5, in getOneUrl\n",
      "    r=requests.get(url,headers=headers,timeout=30)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 72, in get\n",
      "    return request('get', url, params=params, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/api.py\", line 58, in request\n",
      "    return session.request(method=method, url=url, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 508, in request\n",
      "    resp = self.send(prep, **send_kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/sessions.py\", line 618, in send\n",
      "    r = adapter.send(request, **kwargs)\n",
      "  File \"/home/lyndonlcf/anaconda3/lib/python3.6/site-packages/requests/adapters.py\", line 508, in send\n",
      "    raise ConnectionError(e, request=request)\n",
      "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='www.adidas.com.cn', port=443): Max retries exceeded with url: /location/storelist.json?startPage=1&size=12&province=%E7%A6%8F%E5%BB%BA&city=%E6%B3%89%E5%B7%9E%E5%B8%82&storeName=&types%5B%5D=%E7%AB%A5%E8%A3%85%E5%BA%97&_=1556376900755 (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f3ad0905908>: Failed to establish a new connection: [Errno -2] Name or service not known',))\n"
     ]
    }
   ],
   "source": [
    "cc=0\n",
    "for pkey,pval in province.items():\n",
    "    prtt=['','','','']\n",
    "    try:\n",
    "        cc+=1\n",
    "        pv=pval.replace('省','')\n",
    "        prtt[0]=pv\n",
    "        print(pv,cc,end=' ')\n",
    "        res[pv]={}\n",
    "        purl='https://www.adidas.com.cn/location/city.json?province={0}&_=1556377173555'.format(pv)\n",
    "        cts=getOneUrl(purl)\n",
    "        if cts==None:\n",
    "            print('cts==None',pv,purl)\n",
    "            cts='[]'\n",
    "        citys=json.loads(cts)\n",
    "        for c in citys:\n",
    "            res[pv][c]={}\n",
    "            prtt[1]=c\n",
    "            sts=getStoreTypes(c) #sotretypes list str\n",
    "            for s in sts:\n",
    "                prtt[2]=s\n",
    "                stotal=getSTotal(pv,c,s)\n",
    "                res[pv][c][s]=stotal[0]\n",
    "                klst.extend(stotal[1])\n",
    "                prtt[3]=stotal[0]\n",
    "                #time.sleep(1)\n",
    "            time.sleep(1)\n",
    "            \n",
    "    except:\n",
    "        traceback.print_exc()\n",
    "        print(prtt)\n",
    "        #break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#https://www.adidas.com.cn/location/storefinder?locale=zh_CN"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'河北': {'安国市': {'运动表现店': 1},\n",
       "  '安平县': {'运动表现店': 1},\n",
       "  '安新县': {'运动表现店': 2, '运动休闲店': 1}}}"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10101\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'addressCN': '河北安国市药华大街170号宜家旺商场4楼',\n",
       " 'addressEN': '河北安国市药华大街170号宜家旺商场4楼',\n",
       " 'city': '安国市',\n",
       " 'id': 5351,\n",
       " 'latitude': '38.422413',\n",
       " 'longitude': '115.321785',\n",
       " 'pOSNameCN': '安国市药华大街宜家旺商场运动表现店',\n",
       " 'pOSNameEN': '安国市药华大街宜家旺商场运动表现店',\n",
       " 'province': '河北',\n",
       " 'storeType': '运动表现店',\n",
       " 'storeTypeEN': 'Sport Performance',\n",
       " 'telephone': '18832238828',\n",
       " 'workHourCN': '',\n",
       " 'workHourEn': ''}"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print(len(klst))  #需要去重\n",
    "klst[0]  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'河北': {'安国市': {'运动表现店': 1},\n",
       "  '安平县': {'运动表现店': 1},\n",
       "  '安新县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '霸州市': {'运动表现店': 4},\n",
       "  '保定市': {'运动表现店': 30, '三叶草店': 30, '运动休闲店': 30, '童装店': 30, '折扣店': 30},\n",
       "  '泊头市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '沧州市': {'运动表现店': 14, '运动休闲店': 14, '童装店': 14},\n",
       "  '昌黎县': {'运动表现店': 2},\n",
       "  '承德市': {'运动表现店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '磁县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '大城县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '大名县': {'运动表现店': 1},\n",
       "  '定兴县': {'运动表现店': 2},\n",
       "  '定州市': {'运动表现店': 8, '运动休闲店': 8},\n",
       "  '东光县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '丰宁满族自治县': {'运动表现店': 1},\n",
       "  '抚宁县': {'运动表现店': 1},\n",
       "  '阜城县': {'运动表现店': 1},\n",
       "  '高碑店市': {'运动表现店': 8, '运动休闲店': 8},\n",
       "  '高阳县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '故城县': {'运动表现店': 1},\n",
       "  '固安县': {'运动表现店': 3},\n",
       "  '馆陶县': {'运动表现店': 1},\n",
       "  '广宗县': {'运动表现店': 1},\n",
       "  '海兴县': {'运动表现店': 1},\n",
       "  '邯郸市': {'运动表现店': 18, '三叶草店': 18, '运动休闲店': 18, '童装店': 18, '折扣店': 18},\n",
       "  '河间市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '衡水市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '怀来县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '黄骅市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '冀州市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '晋州市': {'运动表现店': 2},\n",
       "  '景县': {'运动表现店': 1},\n",
       "  '宽城满族自治县': {'运动表现店': 1},\n",
       "  '廊坊市': {'运动表现店': 15, '三叶草店': 15, '运动休闲店': 15, '童装店': 15},\n",
       "  '乐亭县': {'运动表现店': 3},\n",
       "  '临漳县': {'运动表现店': 1},\n",
       "  '灵寿县': {'运动表现店': 1},\n",
       "  '隆化县': {'运动表现店': 1},\n",
       "  '卢龙县': {'运动表现店': 1},\n",
       "  '鹿泉市': {'运动表现店': 2, '童装店': 2},\n",
       "  '滦南县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '滦县': {'运动表现店': 2},\n",
       "  '满城县': {'运动表现店': 1},\n",
       "  '南宫市': {'运动表现店': 1},\n",
       "  '南皮县': {'运动表现店': 1},\n",
       "  '内丘县': {'运动表现店': 1},\n",
       "  '宁晋县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '平山县': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '迁安市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '迁西县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '秦皇岛市': {'运动表现店': 28, '三叶草店': 28, '运动休闲店': 28, '童装店': 28},\n",
       "  '青县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '清河县': {'运动表现店': 2},\n",
       "  '清苑县': {'运动表现店': 1},\n",
       "  '曲阳县': {'运动表现店': 2},\n",
       "  '曲周县': {'运动表现店': 1},\n",
       "  '饶阳县': {'运动表现店': 1},\n",
       "  '任丘市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '任县': {'运动表现店': 1},\n",
       "  '容城县': {'运动表现店': 2},\n",
       "  '三河市': {'运动表现店': 16, '运动休闲店': 16, '童装店': 16},\n",
       "  '沙河市': {'运动表现店': 1},\n",
       "  '涉县': {'运动表现店': 1},\n",
       "  '深泽县': {'运动表现店': 1},\n",
       "  '深州市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '石家庄市': {'迷你品牌中心': 65,\n",
       "   '运动表现店': 65,\n",
       "   '三叶草店': 65,\n",
       "   '运动休闲店': 65,\n",
       "   '篮球店': 65,\n",
       "   '童装店': 65,\n",
       "   '折扣店': 65},\n",
       "  '顺平县': {'运动表现店': 1},\n",
       "  '肃宁县': {'运动表现店': 2},\n",
       "  '唐山市': {'运动表现店': 51, '三叶草店': 51, '运动休闲店': 51, '童装店': 51, '折扣店': 51},\n",
       "  '唐县': {'运动表现店': 1},\n",
       "  '望都县': {'运动表现店': 2},\n",
       "  '威县': {'运动表现店': 1},\n",
       "  '围场满族蒙古族自治县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '蔚县': {'运动表现店': 1},\n",
       "  '魏县': {'运动表现店': 1},\n",
       "  '文安县': {'运动表现店': 3},\n",
       "  '无极县': {'运动表现店': 1},\n",
       "  '武强县': {'运动表现店': 1},\n",
       "  '武邑县': {'运动表现店': 1},\n",
       "  '献县': {'运动表现店': 2},\n",
       "  '香河县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '辛集市': {'运动表现店': 3},\n",
       "  '新乐市': {'运动表现店': 1},\n",
       "  '兴隆县': {'运动表现店': 1},\n",
       "  '邢台市': {'运动表现店': 20, '三叶草店': 20, '运动休闲店': 20, '童装店': 20},\n",
       "  '行唐县': {'运动表现店': 1},\n",
       "  '雄县': {'运动表现店': 2},\n",
       "  '徐水县': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '盐山县': {'运动表现店': 1},\n",
       "  '易县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '永年县': {'运动表现店': 1},\n",
       "  '玉田县': {'运动表现店': 4, '童装店': 4},\n",
       "  '元氏县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '赞皇县': {'运动表现店': 2},\n",
       "  '枣强县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '张家口市': {'运动表现店': 17, '运动休闲店': 17, '童装店': 17, '折扣店': 17},\n",
       "  '赵县': {'运动表现店': 1},\n",
       "  '正定县': {'运动表现店': 2},\n",
       "  '遵化市': {'运动表现店': 6, '运动休闲店': 6},\n",
       "  '藁城市': {'运动表现店': 1},\n",
       "  '涞水县': {'运动表现店': 1},\n",
       "  '涞源县': {'运动表现店': 1},\n",
       "  '涿州市': {'运动表现店': 9, '运动休闲店': 9},\n",
       "  '栾城县': {'运动表现店': 2},\n",
       "  '蠡县': {'运动表现店': 1}},\n",
       " '西藏': {'昌都县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '拉萨市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '林芝县': {'运动表现店': 1},\n",
       "  '那曲县': {'运动表现店': 1},\n",
       "  '乃东县': {'运动表现店': 1},\n",
       "  '日喀则市': {'运动表现店': 1}},\n",
       " '浙江': {'安吉县': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '苍南县': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '常山县': {'运动表现店': 1},\n",
       "  '长兴县': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '淳安县': {'运动表现店': 2},\n",
       "  '慈溪市': {'运动表现店': 19, '三叶草店': 19, '运动休闲店': 19, '童装店': 19, '折扣店': 19},\n",
       "  '德清县': {'运动表现店': 4, '童装店': 4},\n",
       "  '东阳市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '奉化市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '富阳市': {'运动表现店': 11, '三叶草店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '海宁市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '海盐县': {'运动表现店': 3},\n",
       "  '杭州市': {'迷你品牌中心': 164,\n",
       "   '运动表现店': 164,\n",
       "   '三叶草店': 164,\n",
       "   '运动休闲店': 164,\n",
       "   '篮球店': 164,\n",
       "   '童装店': 164,\n",
       "   '箱包店': 164,\n",
       "   '折扣店': 164},\n",
       "  '湖州市': {'运动表现店': 23, '三叶草店': 23, '运动休闲店': 23, '童装店': 23, '折扣店': 23},\n",
       "  '嘉善县': {'运动表现店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '嘉兴市': {'运动表现店': 21, '三叶草店': 21, '运动休闲店': 21, '童装店': 21},\n",
       "  '建德市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '江山市': {'运动表现店': 1},\n",
       "  '金华市': {'运动表现店': 20, '三叶草店': 20, '运动休闲店': 20, '童装店': 20, '折扣店': 20},\n",
       "  '开化县': {'运动表现店': 2},\n",
       "  '兰溪市': {'运动表现店': 4, '童装店': 4},\n",
       "  '乐清市': {'运动表现店': 13, '三叶草店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '丽水市': {'运动表现店': 10, '三叶草店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '临安市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '临海市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '龙泉市': {'运动表现店': 1},\n",
       "  '龙游县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '宁波市': {'迷你品牌中心': 62,\n",
       "   '运动表现店': 62,\n",
       "   '三叶草店': 62,\n",
       "   '运动休闲店': 62,\n",
       "   '童装店': 62,\n",
       "   '箱包店': 62,\n",
       "   '折扣店': 62},\n",
       "  '宁海县': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '磐安县': {'运动表现店': 1},\n",
       "  '平湖市': {'运动表现店': 2, '童装店': 2},\n",
       "  '平阳县': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '浦江县': {'运动表现店': 2, '童装店': 2},\n",
       "  '青田县': {'运动表现店': 1},\n",
       "  '庆元县': {'运动表现店': 1},\n",
       "  '瑞安市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '三门县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '上虞市': {'运动表现店': 11, '三叶草店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '绍兴市': {'运动表现店': 22, '三叶草店': 22, '运动休闲店': 22, '童装店': 22, '折扣店': 22},\n",
       "  '绍兴县': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '遂昌县': {'运动表现店': 1},\n",
       "  '台州市': {'运动表现店': 30,\n",
       "   '三叶草店': 30,\n",
       "   '运动休闲店': 30,\n",
       "   '童装店': 30,\n",
       "   '箱包店': 30,\n",
       "   '折扣店': 30},\n",
       "  '泰顺县': {'运动表现店': 2, '童装店': 2},\n",
       "  '天台县': {'运动表现店': 2, '童装店': 2},\n",
       "  '桐庐县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '桐乡市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '温岭市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '温州市': {'迷你品牌中心': 28,\n",
       "   '运动休闲概念店': 28,\n",
       "   '运动表现店': 28,\n",
       "   '三叶草店': 28,\n",
       "   '运动休闲店': 28,\n",
       "   '童装店': 28,\n",
       "   '折扣店': 28},\n",
       "  '文成县': {'运动表现店': 1},\n",
       "  '武义县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '仙居县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '象山县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '新昌县': {'运动表现店': 3},\n",
       "  '义乌市': {'运动表现店': 21,\n",
       "   '三叶草店': 21,\n",
       "   '运动休闲店': 21,\n",
       "   '童装店': 21,\n",
       "   '箱包店': 21,\n",
       "   '折扣店': 21},\n",
       "  '永嘉县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '永康市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '余姚市': {'运动表现店': 16, '三叶草店': 16, '运动休闲店': 16, '童装店': 16, '折扣店': 16},\n",
       "  '玉环县': {'运动表现店': 4, '童装店': 4},\n",
       "  '云和县': {'运动表现店': 1},\n",
       "  '舟山市': {'运动表现店': 9, '运动休闲店': 9},\n",
       "  '诸暨市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '岱山县': {'运动表现店': 1},\n",
       "  '嵊州市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '衢州市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '缙云县': {'运动表现店': 3}},\n",
       " '甘肃': {'白银市': {'运动表现店': 1},\n",
       "  '定西市': {'运动表现店': 2},\n",
       "  '敦煌市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '合作市': {'运动表现店': 1},\n",
       "  '华亭县': {'运动表现店': 2},\n",
       "  '环县': {'运动表现店': 1},\n",
       "  '嘉峪关市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '金昌市': {'运动表现店': 2},\n",
       "  '景泰县': {'运动表现店': 1},\n",
       "  '酒泉市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '兰州市': {'运动表现店': 36, '三叶草店': 36, '运动休闲店': 36, '童装店': 36, '箱包店': 36},\n",
       "  '临夏县': {'运动表现店': 1},\n",
       "  '临泽县': {'运动表现店': 1},\n",
       "  '陇南市': {'运动表现店': 1},\n",
       "  '陇西县': {'运动表现店': 1},\n",
       "  '玛曲县': {'运动表现店': 1},\n",
       "  '民乐县': {'运动表现店': 1},\n",
       "  '平凉市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '庆城县': {'运动表现店': 1},\n",
       "  '庆阳市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '天水市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '武威市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '张掖市': {'运动表现店': 3, '运动休闲店': 3}},\n",
       " '江西': {'东乡县': {'运动表现店': 1},\n",
       "  '抚州市': {'运动表现店': 3},\n",
       "  '赣州市': {'运动表现店': 7, '三叶草店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '高安市': {'运动表现店': 1},\n",
       "  '广丰县': {'运动表现店': 1},\n",
       "  '贵溪市': {'运动表现店': 1},\n",
       "  '吉安市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '进贤县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '景德镇市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '九江市': {'运动表现店': 17, '三叶草店': 17, '运动休闲店': 17, '童装店': 17, '折扣店': 17},\n",
       "  '南昌市': {'运动表现店': 64, '三叶草店': 64, '运动休闲店': 64, '童装店': 64, '折扣店': 64},\n",
       "  '南昌县': {'运动表现店': 2},\n",
       "  '南康市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '萍乡市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '瑞昌市': {'运动表现店': 1},\n",
       "  '上高县': {'运动表现店': 1},\n",
       "  '上饶市': {'运动表现店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '新建县': {'运动表现店': 2},\n",
       "  '新余市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '修水县': {'运动表现店': 1},\n",
       "  '宜春市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '鹰潭市': {'运动表现店': 2},\n",
       "  '余干县': {'运动表现店': 1},\n",
       "  '玉山县': {'运动表现店': 1},\n",
       "  '鄱阳县': {'运动表现店': 1}},\n",
       " '辽宁': {'鞍山市': {'运动表现店': 25,\n",
       "   '三叶草店': 25,\n",
       "   '运动休闲店': 25,\n",
       "   '篮球店': 25,\n",
       "   '童装店': 25,\n",
       "   '箱包店': 25,\n",
       "   '折扣店': 25},\n",
       "  '北镇市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '本溪满族自治县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '本溪市': {'运动表现店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '昌图县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '朝阳市': {'运动表现店': 14, '运动休闲店': 14, '童装店': 14},\n",
       "  '大连市': {'迷你品牌中心': 121,\n",
       "   '运动休闲概念店': 121,\n",
       "   '运动表现店': 121,\n",
       "   '三叶草店': 121,\n",
       "   '运动休闲店': 121,\n",
       "   '足球店': 121,\n",
       "   '篮球店': 121,\n",
       "   '户外店': 121,\n",
       "   '童装店': 121,\n",
       "   '箱包店': 121,\n",
       "   '折扣店': 121},\n",
       "  '大石桥市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '丹东市': {'运动表现店': 15, '三叶草店': 15, '运动休闲店': 15, '童装店': 15, '折扣店': 15},\n",
       "  '灯塔市': {'运动表现店': 1},\n",
       "  '东港市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '法库县': {'运动表现店': 1},\n",
       "  '凤城市': {'运动表现店': 1},\n",
       "  '抚顺市': {'运动表现店': 18,\n",
       "   '三叶草店': 18,\n",
       "   '运动休闲店': 18,\n",
       "   '篮球店': 18,\n",
       "   '童装店': 18,\n",
       "   '折扣店': 18},\n",
       "  '阜新市': {'运动表现店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '盖州市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '海城市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '黑山县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '葫芦岛市': {'运动表现店': 11, '三叶草店': 11, '运动休闲店': 11, '童装店': 11, '折扣店': 11},\n",
       "  '桓仁满族自治县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '建昌县': {'运动表现店': 3, '运动休闲店': 3, '童装店': 3},\n",
       "  '锦州市': {'运动表现店': 16,\n",
       "   '三叶草店': 16,\n",
       "   '运动休闲店': 16,\n",
       "   '户外店': 16,\n",
       "   '童装店': 16,\n",
       "   '箱包店': 16,\n",
       "   '折扣店': 16},\n",
       "  '喀喇沁左翼蒙古族自治县': {'运动表现店': 2},\n",
       "  '开原市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '康平县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '宽甸满族自治县': {'运动表现店': 1},\n",
       "  '辽阳市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '辽阳县': {'运动表现店': 1},\n",
       "  '辽中县': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '凌海市': {'运动表现店': 2, '童装店': 2},\n",
       "  '凌源市': {'运动表现店': 3},\n",
       "  '盘锦市': {'运动表现店': 21, '三叶草店': 21, '运动休闲店': 21, '户外店': 21, '童装店': 21},\n",
       "  '普兰店市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '沈阳市': {'品牌中心': 120,\n",
       "   '迷你品牌中心': 120,\n",
       "   '运动表现店': 120,\n",
       "   '三叶草店': 120,\n",
       "   '运动休闲店': 120,\n",
       "   '篮球店': 120,\n",
       "   '户外店': 120,\n",
       "   '童装店': 120,\n",
       "   '箱包店': 120,\n",
       "   '折扣店': 120},\n",
       "  '绥中县': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '台安县': {'运动表现店': 1},\n",
       "  '铁岭市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '瓦房店市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11, '折扣店': 11},\n",
       "  '西丰县': {'运动表现店': 1},\n",
       "  '新民市': {'运动表现店': 3, '运动休闲店': 3, '童装店': 3},\n",
       "  '兴城市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '义县': {'运动表现店': 1},\n",
       "  '营口市': {'运动表现店': 29, '三叶草店': 29, '运动休闲店': 29, '童装店': 29},\n",
       "  '彰武县': {'运动表现店': 1},\n",
       "  '庄河市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '岫岩满族自治县': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5}},\n",
       " '重庆': {'城口县': {'运动表现店': 1},\n",
       "  '垫江县': {'运动表现店': 1},\n",
       "  '丰都县': {'运动表现店': 2, '童装店': 2},\n",
       "  '奉节县': {'运动表现店': 1},\n",
       "  '开县': {'运动表现店': 3, '运动休闲店': 3, '童装店': 3},\n",
       "  '梁平县': {'运动表现店': 1},\n",
       "  '彭水苗族土家族自治县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '荣昌县': {'运动表现店': 1},\n",
       "  '石柱土家族自治县': {'运动表现店': 1},\n",
       "  '铜梁县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '巫溪县': {'运动表现店': 1},\n",
       "  '武隆县': {'运动表现店': 1},\n",
       "  '秀山土家族苗族自治县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '酉阳土家族苗族自治县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '云阳县': {'运动表现店': 1},\n",
       "  '忠县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '重庆市': {'品牌中心': 185,\n",
       "   '迷你品牌中心': 185,\n",
       "   '运动休闲概念店': 185,\n",
       "   '运动表现店': 185,\n",
       "   '三叶草店': 185,\n",
       "   '运动休闲店': 185,\n",
       "   '童装店': 185,\n",
       "   '箱包店': 185,\n",
       "   '折扣店': 185},\n",
       "  '潼南县': {'运动表现店': 1},\n",
       "  '璧山县': {'运动表现店': 3, '运动休闲店': 3}},\n",
       " '山东': {'安丘市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '滨州市': {'运动表现店': 18, '运动休闲店': 18, '童装店': 18, '箱包店': 18},\n",
       "  '博兴县': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '曹县': {'运动表现店': 2},\n",
       "  '昌乐县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '昌邑市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '长岛县': {'运动表现店': 1},\n",
       "  '成武县': {'运动表现店': 1},\n",
       "  '单县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '德州市': {'运动表现店': 14, '三叶草店': 14, '运动休闲店': 14, '童装店': 14},\n",
       "  '定陶县': {'运动表现店': 1},\n",
       "  '东阿县': {'运动表现店': 1},\n",
       "  '东明县': {'运动表现店': 4},\n",
       "  '东平县': {'运动表现店': 1},\n",
       "  '东营市': {'运动表现店': 39,\n",
       "   '三叶草店': 39,\n",
       "   '运动休闲店': 39,\n",
       "   '童装店': 39,\n",
       "   '箱包店': 39,\n",
       "   '折扣店': 39},\n",
       "  '肥城市': {'运动表现店': 7, '运动休闲店': 7},\n",
       "  '费县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '高密市': {'运动表现店': 7, '运动休闲店': 7},\n",
       "  '高青县': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '高唐县': {'运动表现店': 1},\n",
       "  '冠县': {'运动表现店': 1},\n",
       "  '广饶县': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '海阳市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '菏泽市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '桓台县': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '惠民县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '即墨市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '济南市': {'迷你品牌中心': 87,\n",
       "   '运动表现店': 87,\n",
       "   '三叶草店': 87,\n",
       "   '运动休闲店': 87,\n",
       "   '童装店': 87,\n",
       "   '折扣店': 87},\n",
       "  '济宁市': {'运动表现店': 27, '三叶草店': 27, '运动休闲店': 27, '童装店': 27},\n",
       "  '济阳县': {'运动表现店': 1},\n",
       "  '嘉祥县': {'运动表现店': 2},\n",
       "  '胶南市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '胶州市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '金乡县': {'运动表现店': 1},\n",
       "  '巨野县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '垦利县': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '莱芜市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '莱西市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '莱阳市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '莱州市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '兰陵县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '乐陵市': {'运动表现店': 2},\n",
       "  '利津县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '梁山县': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '聊城市': {'运动表现店': 15, '运动休闲店': 15, '童装店': 15, '箱包店': 15},\n",
       "  '临清市': {'运动表现店': 1},\n",
       "  '临沂市': {'运动表现店': 37, '三叶草店': 37, '运动休闲店': 37, '童装店': 37},\n",
       "  '临邑县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '临沭县': {'运动表现店': 3},\n",
       "  '临朐县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '陵县': {'运动表现店': 2},\n",
       "  '龙口市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '蒙阴县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '宁津县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '蓬莱市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '平度市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '平邑县': {'运动表现店': 3},\n",
       "  '平阴县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '平原县': {'运动表现店': 2},\n",
       "  '栖霞市': {'运动表现店': 1},\n",
       "  '齐河县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '青岛市': {'迷你品牌中心': 115,\n",
       "   '运动表现店': 115,\n",
       "   '三叶草店': 115,\n",
       "   '运动休闲店': 115,\n",
       "   '户外店': 115,\n",
       "   '童装店': 115,\n",
       "   '箱包店': 115,\n",
       "   '折扣店': 115},\n",
       "  '青州市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '庆云县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '曲阜市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '日照市': {'运动表现店': 17, '三叶草店': 17, '运动休闲店': 17, '童装店': 17},\n",
       "  '荣成市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '乳山市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '商河县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '寿光市': {'运动表现店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '泰安市': {'运动表现店': 15, '三叶草店': 15, '运动休闲店': 15, '童装店': 15, '折扣店': 15},\n",
       "  '威海市': {'运动表现店': 22, '三叶草店': 22, '运动休闲店': 22, '童装店': 22, '折扣店': 22},\n",
       "  '微山县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '潍坊市': {'运动表现店': 28, '三叶草店': 28, '运动休闲店': 28, '童装店': 28, '折扣店': 28},\n",
       "  '文登市': {'运动表现店': 7, '运动休闲店': 7},\n",
       "  '无棣县': {'运动表现店': 3},\n",
       "  '五莲县': {'运动表现店': 2},\n",
       "  '夏津县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '新泰市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '烟台市': {'运动表现店': 43, '三叶草店': 43, '运动休闲店': 43, '童装店': 43},\n",
       "  '阳谷县': {'运动表现店': 1},\n",
       "  '阳信县': {'运动表现店': 2},\n",
       "  '沂南县': {'运动表现店': 3},\n",
       "  '沂水县': {'运动表现店': 2},\n",
       "  '沂源县': {'运动表现店': 1},\n",
       "  '鱼台县': {'运动表现店': 1},\n",
       "  '禹城市': {'运动表现店': 3},\n",
       "  '枣庄市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '章丘市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '招远市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '诸城市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '淄博市': {'运动表现店': 50,\n",
       "   '三叶草店': 50,\n",
       "   '运动休闲店': 50,\n",
       "   '童装店': 50,\n",
       "   '箱包店': 50,\n",
       "   '折扣店': 50},\n",
       "  '邹城市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '邹平县': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '兖州市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '郓城县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '郯城县': {'运动表现店': 5},\n",
       "  '茌平县': {'运动表现店': 1},\n",
       "  '莒南县': {'运动表现店': 3},\n",
       "  '莒县': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '莘县': {'运动表现店': 1},\n",
       "  '汶上县': {'运动表现店': 2},\n",
       "  '泗水县': {'运动表现店': 2},\n",
       "  '滕州市': {'运动表现店': 4, '运动休闲店': 4}},\n",
       " '海南': {'海口市': {'迷你品牌中心': 47,\n",
       "   '运动休闲概念店': 47,\n",
       "   '运动表现店': 47,\n",
       "   '三叶草店': 47,\n",
       "   '运动休闲店': 47,\n",
       "   '童装店': 47,\n",
       "   '折扣店': 47},\n",
       "  '陵水黎族自治县': {'运动表现店': 1},\n",
       "  '琼海市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '三亚市': {'运动表现店': 15, '运动休闲店': 15, '童装店': 15, '折扣店': 15},\n",
       "  '万宁市': {'折扣店': 1},\n",
       "  '文昌市': {'运动表现店': 1},\n",
       "  '儋州市': {'运动表现店': 4, '运动休闲店': 4}},\n",
       " '云南': {'安宁市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '保山市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '宾川县': {'运动表现店': 1},\n",
       "  '沧源佤族自治县': {'运动表现店': 1},\n",
       "  '澄江县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '楚雄市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '大理市': {'运动表现店': 11, '三叶草店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '大姚县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '洱源县': {'运动表现店': 1},\n",
       "  '富宁县': {'运动表现店': 1},\n",
       "  '富源县': {'运动表现店': 1},\n",
       "  '个旧市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '耿马傣族佤族自治县': {'运动表现店': 1},\n",
       "  '广南县': {'运动表现店': 1},\n",
       "  '河口瑶族自治县': {'运动表现店': 1},\n",
       "  '鹤庆县': {'运动表现店': 1},\n",
       "  '华宁县': {'运动表现店': 1},\n",
       "  '会泽县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '剑川县': {'运动表现店': 1},\n",
       "  '建水县': {'运动表现店': 5, '童装店': 5},\n",
       "  '江川县': {'运动表现店': 1},\n",
       "  '晋宁县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '景东彝族自治县': {'运动表现店': 1},\n",
       "  '景谷傣族彝族自治县': {'运动表现店': 1},\n",
       "  '景洪市': {'运动表现店': 14, '运动休闲店': 14},\n",
       "  '开远市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '昆明市': {'迷你品牌中心': 131,\n",
       "   '运动休闲概念店': 131,\n",
       "   '运动表现店': 131,\n",
       "   '三叶草店': 131,\n",
       "   '运动休闲店': 131,\n",
       "   '童装店': 131,\n",
       "   '箱包店': 131,\n",
       "   '折扣店': 131},\n",
       "  '兰坪白族普米族自治县': {'运动表现店': 1},\n",
       "  '澜沧拉祜族自治县': {'运动表现店': 1},\n",
       "  '丽江市': {'运动表现店': 20}},\n",
       " '贵州': {'安顺市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '毕节市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '赤水市': {'运动表现店': 2, '童装店': 2},\n",
       "  '德江县': {'运动表现店': 1},\n",
       "  '都匀市': {'运动表现店': 2},\n",
       "  '福泉市': {'运动表现店': 1},\n",
       "  '贵定县': {'运动表现店': 1},\n",
       "  '贵阳市': {'迷你品牌中心': 78,\n",
       "   '运动表现店': 78,\n",
       "   '三叶草店': 78,\n",
       "   '运动休闲店': 78,\n",
       "   '篮球店': 78,\n",
       "   '童装店': 78,\n",
       "   '箱包店': 78,\n",
       "   '折扣店': 78},\n",
       "  '金沙县': {'运动表现店': 1},\n",
       "  '开阳县': {'运动表现店': 1},\n",
       "  '凯里市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '六盘水市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '盘县': {'运动表现店': 1},\n",
       "  '平坝县': {'运动表现店': 1},\n",
       "  '黔西县': {'运动表现店': 1},\n",
       "  '仁怀市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '石阡县': {'运动表现店': 1},\n",
       "  '思南县': {'运动表现店': 1},\n",
       "  '铜仁市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '威宁彝族回族苗族自治县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '习水县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '兴义市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '印江土家族苗族自治县': {'运动表现店': 1},\n",
       "  '正安县': {'运动表现店': 1},\n",
       "  '织金县': {'运动表现店': 1},\n",
       "  '遵义市': {'运动表现店': 24, '三叶草店': 24, '运动休闲店': 24, '童装店': 24},\n",
       "  '湄潭县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '榕江县': {'运动表现店': 1}},\n",
       " '安徽': {'安庆市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '蚌埠市': {'运动表现店': 26,\n",
       "   '三叶草店': 26,\n",
       "   '运动休闲店': 26,\n",
       "   '篮球店': 26,\n",
       "   '童装店': 26,\n",
       "   '折扣店': 26},\n",
       "  '长丰县': {'运动表现店': 1},\n",
       "  '巢湖市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '池州市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '滁州市': {'运动表现店': 6, '运动休闲店': 6},\n",
       "  '当涂县': {'运动表现店': 1},\n",
       "  '繁昌县': {'运动表现店': 1},\n",
       "  '肥东县': {'运动表现店': 1},\n",
       "  '肥西县': {'运动表现店': 2, '童装店': 2},\n",
       "  '凤台县': {'运动表现店': 1},\n",
       "  '凤阳县': {'运动表现店': 1},\n",
       "  '阜南县': {'运动表现店': 1},\n",
       "  '阜阳市': {'运动表现店': 16, '运动休闲店': 16, '童装店': 16, '折扣店': 16},\n",
       "  '固镇县': {'运动表现店': 1},\n",
       "  '广德县': {'运动表现店': 2},\n",
       "  '和县': {'运动表现店': 2},\n",
       "  '合肥市': {'迷你品牌中心': 104,\n",
       "   '运动表现店': 104,\n",
       "   '三叶草店': 104,\n",
       "   '运动休闲店': 104,\n",
       "   '篮球店': 104,\n",
       "   '户外店': 104,\n",
       "   '童装店': 104,\n",
       "   '箱包店': 104,\n",
       "   '折扣店': 104},\n",
       "  '怀宁县': {'运动表现店': 1},\n",
       "  '怀远县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '淮北市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '淮南市': {'运动表现店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '黄山市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '霍邱县': {'运动表现店': 1},\n",
       "  '界首市': {'运动表现店': 1},\n",
       "  '郎溪县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '临泉县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '灵璧县': {'运动表现店': 1},\n",
       "  '六安市': {'运动表现店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '庐江县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '马鞍山市': {}},\n",
       " '吉林': {'安图县': {'运动表现店': 1},\n",
       "  '白城市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '白山市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '长白朝鲜族自治县': {'运动表现店': 1},\n",
       "  '长春市': {'迷你品牌中心': 79,\n",
       "   '运动休闲概念店': 79,\n",
       "   '运动表现店': 79,\n",
       "   '三叶草店': 79,\n",
       "   '运动休闲店': 79,\n",
       "   '户外店': 79,\n",
       "   '童装店': 79,\n",
       "   '折扣店': 79},\n",
       "  '大安市': {'运动表现店': 1},\n",
       "  '德惠市': {'运动表现店': 6, '运动休闲店': 6},\n",
       "  '东丰县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '敦化市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '扶余县': {'运动表现店': 1},\n",
       "  '抚松县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '公主岭市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '辉南县': {'运动表现店': 1},\n",
       "  '吉林市': {'运动表现店': 36, '三叶草店': 36, '运动休闲店': 36, '童装店': 36, '折扣店': 36},\n",
       "  '集安市': {'运动表现店': 1},\n",
       "  '九台市': {'运动表现店': 1},\n",
       "  '辽源市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '临江市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '柳河县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '梅河口市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '农安县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '磐石市': {'运动表现店': 1},\n",
       "  '前郭尔罗斯蒙古族自治县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '舒兰市': {'运动表现店': 1},\n",
       "  '双辽市': {'运动表现店': 2},\n",
       "  '四平市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '松原市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '通化市': {'运动表现店': 7, '运动休闲店': 7, '户外店': 7, '童装店': 7, '折扣店': 7},\n",
       "  '汪清县': {'运动表现店': 1},\n",
       "  '延吉市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '伊通满族自治县': {'运动表现店': 1},\n",
       "  '榆树市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '镇赉县': {'运动表现店': 1},\n",
       "  '洮南市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '珲春市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '桦甸市': {'运动表现店': 5, '运动休闲店': 5}},\n",
       " '青海': {'格尔木市': {'运动表现店': 1},\n",
       "  '西宁市': {'运动表现店': 31, '三叶草店': 31, '运动休闲店': 31, '童装店': 31, '箱包店': 31}},\n",
       " '四川': {'阿坝县': {'运动表现店': 1},\n",
       "  '巴中市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '成都市': {'品牌中心': 174,\n",
       "   '迷你品牌中心': 174,\n",
       "   '运动休闲概念店': 174,\n",
       "   '运动表现店': 174,\n",
       "   '三叶草店': 174,\n",
       "   '运动休闲店': 174,\n",
       "   '跑步旗舰店': 174,\n",
       "   '跑步店': 174,\n",
       "   '篮球店': 174,\n",
       "   '女子店': 174,\n",
       "   '童装店': 174,\n",
       "   '箱包店': 174,\n",
       "   '折扣店': 174},\n",
       "  '崇州市': {'运动表现店': 2},\n",
       "  '达州市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '大邑县': {'运动表现店': 1},\n",
       "  '大竹县': {'运动表现店': 1},\n",
       "  '丹棱县': {'运动表现店': 1},\n",
       "  '德阳市': {'运动表现店': 16, '三叶草店': 16, '运动休闲店': 16, '童装店': 16, '箱包店': 16},\n",
       "  '都江堰市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '峨眉山市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '富顺县': {'运动表现店': 1},\n",
       "  '甘孜县': {'运动表现店': 1},\n",
       "  '古蔺县': {'运动表现店': 1},\n",
       "  '广安市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '广汉市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '广元市': {'运动表现店': 8, '三叶草店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '汉源县': {'运动表现店': 1},\n",
       "  '洪雅县': {'运动表现店': 2},\n",
       "  '会理县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '夹江县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '简阳市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '江油市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '金堂县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '康定县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '乐山市': {'运动表现店': 17, '运动休闲店': 17, '童装店': 17},\n",
       "  '邻水县': {'运动表现店': 1},\n",
       "  '隆昌县': {'运动表现店': 1},\n",
       "  '眉山市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '绵阳市': {'运动表现店': 23, '三叶草店': 23, '运动休闲店': 23, '童装店': 23, '箱包店': 23},\n",
       "  '绵竹市': {'运动表现店': 1},\n",
       "  '名山县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '南部县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '南充市': {'运动表现店': 13, '三叶草店': 13, '运动休闲店': 13, '童装店': 13, '箱包店': 13},\n",
       "  '内江市': {'运动表现店': 11, '三叶草店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '宁南县': {'运动表现店': 1},\n",
       "  '攀枝花市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '彭州市': {'运动表现店': 4, '运动休闲店': 4, '折扣店': 4},\n",
       "  '平昌县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '蒲江县': {'运动表现店': 1},\n",
       "  '青神县': {'运动表现店': 1},\n",
       "  '仁寿县': {'运动表现店': 1},\n",
       "  '若尔盖县': {'运动表现店': 1},\n",
       "  '三台县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '射洪县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '什邡市': {'运动表现店': 3},\n",
       "  '双流县': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '遂宁市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '天全县': {'运动表现店': 1},\n",
       "  '通江县': {'运动表现店': 1},\n",
       "  '旺苍县': {'运动表现店': 1},\n",
       "  '西昌市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '新津县': {'运动表现店': 2},\n",
       "  '雅安市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '宜宾市': {'运动表现店': 16, '运动休闲店': 16, '童装店': 16},\n",
       "  '中江县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '资阳市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '资中县': {'运动表现店': 1},\n",
       "  '自贡市': {'运动表现店': 16, '运动休闲店': 16, '箱包店': 16, '折扣店': 16},\n",
       "  '邛崃市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '郫县': {'运动表现店': 12, '三叶草店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '阆中市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '泸州市': {'运动表现店': 19, '三叶草店': 19, '运动休闲店': 19, '童装店': 19, '箱包店': 19},\n",
       "  '梓潼县': {'运动表现店': 1},\n",
       "  '犍为县': {'运动表现店': 1}},\n",
       " '内蒙古': {'阿拉善左旗': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '阿鲁科尔沁旗': {'运动表现店': 2},\n",
       "  '阿荣旗': {'运动表现店': 1},\n",
       "  '敖汉旗': {'运动表现店': 1},\n",
       "  '巴林右旗': {'运动表现店': 1},\n",
       "  '巴彦淖尔市': {'运动表现店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '包头市': {'运动表现店': 42, '三叶草店': 42, '运动休闲店': 42, '童装店': 42, '折扣店': 42},\n",
       "  '察哈尔右翼后旗': {'运动表现店': 1},\n",
       "  '赤峰市': {'运动表现店': 25, '运动休闲店': 25, '童装店': 25},\n",
       "  '达拉特旗': {'运动表现店': 2},\n",
       "  '鄂尔多斯市': {'运动表现店': 15, '三叶草店': 15, '运动休闲店': 15, '童装店': 15},\n",
       "  '鄂伦春自治旗': {'运动表现店': 1},\n",
       "  '鄂托克前旗': {'运动表现店': 1},\n",
       "  '二连浩特市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '杭锦后旗': {'运动表现店': 2},\n",
       "  '呼和浩特市': {'运动表现店': 48, '三叶草店': 48, '运动休闲店': 48, '童装店': 48, '折扣店': 48},\n",
       "  '呼伦贝尔市': {'运动表现店': 16, '运动休闲店': 16, '童装店': 16},\n",
       "  '霍林郭勒市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '集宁市': {'运动休闲店': 1},\n",
       "  '开鲁县': {'运动表现店': 1},\n",
       "  '科尔沁右翼中旗': {'运动表现店': 1},\n",
       "  '科尔沁左翼后旗': {'运动表现店': 1},\n",
       "  '科尔沁左翼中旗': {'运动表现店': 1},\n",
       "  '克什克腾旗': {'运动表现店': 2},\n",
       "  '库伦旗': {'运动表现店': 1},\n",
       "  '林西县': {'运动表现店': 1},\n",
       "  '满洲里市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '莫力达瓦达斡尔族自治旗': {'运动表现店': 1},\n",
       "  '奈曼旗': {'运动表现店': 1},\n",
       "  '宁城县': {'运动表现店': 1},\n",
       "  '商都县': {'运动表现店': 1},\n",
       "  '四子王旗': {'运动表现店': 1},\n",
       "  '苏尼特右旗': {'运动表现店': 1},\n",
       "  '苏尼特左旗': {'运动表现店': 1},\n",
       "  '通辽市': {'运动表现店': 15, '三叶草店': 15, '运动休闲店': 15, '童装店': 15},\n",
       "  '突泉县': {'运动表现店': 1},\n",
       "  '土默特右旗': {'运动表现店': 1},\n",
       "  '土默特左旗': {'运动表现店': 2},\n",
       "  '托克托县': {'运动表现店': 1},\n",
       "  '乌海市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '乌拉特前旗': {'运动表现店': 3},\n",
       "  '乌拉特中旗': {'运动表现店': 1},\n",
       "  '乌兰察布市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '乌兰浩特市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '乌审旗': {'运动表现店': 1},\n",
       "  '五原县': {'运动表现店': 1},\n",
       "  '西乌珠穆沁旗': {'运动表现店': 1},\n",
       "  '锡林浩特市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '牙克石市': {'运动表现店': 1},\n",
       "  '伊金霍洛旗': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '扎兰屯市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '扎鲁特旗': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '扎赉特旗': {'运动表现店': 2},\n",
       "  '准格尔旗': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '磴口县': {'运动表现店': 1}},\n",
       " '湖北': {'赤壁市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '大冶市': {'运动表现店': 2},\n",
       "  '丹江口市': {'运动表现店': 1},\n",
       "  '鄂州市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '恩施市': {'运动表现店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '汉川市': {'运动表现店': 1},\n",
       "  '洪湖市': {'运动表现店': 1},\n",
       "  '黄冈市': {'运动表现店': 8, '运动休闲店': 8},\n",
       "  '黄梅县': {'运动表现店': 1},\n",
       "  '黄石市': {'运动表现店': 14, '三叶草店': 14, '运动休闲店': 14, '童装店': 14},\n",
       "  '监利县': {'运动表现店': 2},\n",
       "  '荆门市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '荆州市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '京山县': {'运动表现店': 1},\n",
       "  '来凤县': {'运动表现店': 1},\n",
       "  '老河口市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '利川市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '罗田县': {'运动表现店': 1},\n",
       "  '麻城市': {'运动表现店': 1},\n",
       "  '潜江市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '十堰市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '石首市': {'运动表现店': 1},\n",
       "  '随州市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '天门市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '通城县': {'运动表现店': 1},\n",
       "  '武汉市': {'品牌中心': 130,\n",
       "   '迷你品牌中心': 130,\n",
       "   '运动表现店': 130,\n",
       "   '三叶草店': 130,\n",
       "   '运动休闲店': 130,\n",
       "   '户外店': 130,\n",
       "   '童装店': 130,\n",
       "   '折扣店': 130},\n",
       "  '武穴市': {'运动表现店': 1},\n",
       "  '仙桃市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '咸宁市': {'运动休闲店': 1},\n",
       "  '襄阳市': {'运动表现店': 18, '运动休闲店': 18, '童装店': 18},\n",
       "  '孝感市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '阳新县': {'运动表现店': 1},\n",
       "  '宜昌市': {'运动表现店': 15, '运动休闲店': 15, '童装店': 15},\n",
       "  '宜城市': {'运动表现店': 1},\n",
       "  '应城市': {'运动表现店': 1},\n",
       "  '枝江市': {'运动表现店': 1},\n",
       "  '钟祥市': {'运动表现店': 2},\n",
       "  '蕲春县': {'运动表现店': 1}},\n",
       " '宁夏': {'固原市': {'运动表现店': 1},\n",
       "  '石嘴山市': {'运动表现店': 1},\n",
       "  '同心县': {'运动表现店': 1},\n",
       "  '吴忠市': {'运动表现店': 1},\n",
       "  '银川市': {'运动表现店': 29,\n",
       "   '三叶草店': 29,\n",
       "   '运动休闲店': 29,\n",
       "   '童装店': 29,\n",
       "   '箱包店': 29,\n",
       "   '折扣店': 29},\n",
       "  '中卫市': {'运动表现店': 4}},\n",
       " '广西': {'百色市': {'运动表现店': 3},\n",
       "  '北海市': {'运动表现店': 7, '运动休闲店': 7},\n",
       "  '宾阳县': {'运动表现店': 1},\n",
       "  '博白县': {'运动表现店': 1},\n",
       "  '崇左市': {'运动表现店': 2},\n",
       "  '东兴市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '防城港市': {'运动表现店': 2},\n",
       "  '扶绥县': {'运动表现店': 2},\n",
       "  '桂林市': {'运动表现店': 24, '三叶草店': 24, '运动休闲店': 24, '童装店': 24},\n",
       "  '桂平市': {'运动表现店': 1},\n",
       "  '贵港市': {'运动表现店': 3, '运动休闲店': 3, '童装店': 3},\n",
       "  '合浦县': {'运动表现店': 1},\n",
       "  '河池市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '贺州市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '横县': {'运动表现店': 1},\n",
       "  '靖西县': {'运动表现店': 1},\n",
       "  '来宾市': {'运动表现店': 2},\n",
       "  '荔浦县': {'运动表现店': 1},\n",
       "  '临桂县': {'运动表现店': 1},\n",
       "  '灵川县': {'运动表现店': 1},\n",
       "  '柳州市': {'运动表现店': 20, '三叶草店': 20, '运动休闲店': 20, '童装店': 20},\n",
       "  '龙胜各族自治县': {'运动表现店': 2},\n",
       "  '鹿寨县': {'运动表现店': 1},\n",
       "  '南宁市': {'运动表现店': 53, '三叶草店': 53, '运动休闲店': 53, '童装店': 53, '折扣店': 53},\n",
       "  '平果县': {'运动表现店': 1},\n",
       "  '平乐县': {'运动表现店': 1},\n",
       "  '钦州市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '全州县': {'运动表现店': 1},\n",
       "  '藤县': {'运动表现店': 1},\n",
       "  '田阳县': {'运动表现店': 1},\n",
       "  '梧州市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '武鸣县': {'运动表现店': 1},\n",
       "  '兴安县': {'运动表现店': 1},\n",
       "  '阳朔县': {'运动表现店': 3},\n",
       "  '宜州市': {'运动表现店': 1},\n",
       "  '玉林市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5}},\n",
       " '新疆': {'阿克苏市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '阿拉尔市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '阿勒泰市': {}},\n",
       " '上海': {'崇明县': {'运动表现店': 2},\n",
       "  '上海市': {'品牌中心': 324,\n",
       "   '迷你品牌中心': 324,\n",
       "   '运动休闲概念店': 324,\n",
       "   '运动表现店': 324,\n",
       "   '三叶草店': 324,\n",
       "   '运动休闲店': 324,\n",
       "   '跑步旗舰店': 324,\n",
       "   '鞋店': 324,\n",
       "   '女子店': 324,\n",
       "   '童装店': 324,\n",
       "   '箱包店': 324,\n",
       "   '折扣店': 324}},\n",
       " '黑龙江': {'安达市': {'运动表现店': 2},\n",
       "  '巴彦县': {'运动表现店': 1},\n",
       "  '拜泉县': {'运动表现店': 1},\n",
       "  '宝清县': {'运动表现店': 1},\n",
       "  '北安市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '大庆市': {'运动表现店': 26, '三叶草店': 26, '运动休闲店': 26, '童装店': 26},\n",
       "  '东宁县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '杜尔伯特蒙古族自治县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '方正县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '富锦市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '富裕县': {'运动表现店': 1},\n",
       "  '哈尔滨市': {'迷你品牌中心': 108,\n",
       "   '运动表现店': 108,\n",
       "   '三叶草店': 108,\n",
       "   '运动休闲店': 108,\n",
       "   '篮球店': 108,\n",
       "   '户外店': 108,\n",
       "   '童装店': 108,\n",
       "   '折扣店': 108},\n",
       "  '海伦市': {'运动表现店': 1},\n",
       "  '鹤岗市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '黑河市': {'运动表现店': 3, '运动休闲店': 3, '童装店': 3},\n",
       "  '虎林市': {'运动表现店': 1},\n",
       "  '鸡西市': {'运动表现店': 12, '三叶草店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '集贤县': {'运动表现店': 1},\n",
       "  '佳木斯市': {'运动表现店': 13, '三叶草店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '加格达奇市': {'运动休闲店': 1},\n",
       "  '兰西县': {'运动表现店': 2},\n",
       "  '林口县': {'运动表现店': 1},\n",
       "  '龙江县': {'运动表现店': 1},\n",
       "  '萝北县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '密山市': {'运动表现店': 2},\n",
       "  '牡丹江市': {'运动表现店': 21, '三叶草店': 21, '运动休闲店': 21, '户外店': 21, '童装店': 21},\n",
       "  '木兰县': {'运动表现店': 1},\n",
       "  '嫩江县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '七台河市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '齐齐哈尔市': {'运动表现店': 20, '三叶草店': 20, '运动休闲店': 20, '童装店': 20, '折扣店': 20},\n",
       "  '青冈县': {'运动表现店': 2},\n",
       "  '庆安县': {'运动表现店': 1},\n",
       "  '尚志市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '双城市': {'运动表现店': 1},\n",
       "  '双鸭山市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '绥芬河市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '绥化市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '绥棱县': {'运动表现店': 1},\n",
       "  '铁力市': {'运动表现店': 2},\n",
       "  '通河县': {'运动表现店': 1},\n",
       "  '同江市': {'运动表现店': 1},\n",
       "  '望奎县': {'运动表现店': 1},\n",
       "  '五常市': {'运动表现店': 1},\n",
       "  '延寿县': {'运动表现店': 1},\n",
       "  '依兰县': {'运动表现店': 1},\n",
       "  '伊春市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '友谊县': {'运动表现店': 1},\n",
       "  '肇东市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '肇州县': {'运动表现店': 1},\n",
       "  '讷河市': {'运动表现店': 1},\n",
       "  '桦南县': {'运动表现店': 2}},\n",
       " '河南': {'安阳市': {'运动表现店': 9, '运动休闲店': 9},\n",
       "  '博爱县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '长葛市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '长垣县': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '郸城县': {'运动表现店': 1},\n",
       "  '登封市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '邓州市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '方城县': {'运动表现店': 1},\n",
       "  '封丘县': {'运动表现店': 1},\n",
       "  '扶沟县': {'运动表现店': 1},\n",
       "  '巩义市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '光山县': {'运动表现店': 1},\n",
       "  '鹤壁市': {'运动表现店': 6, '运动休闲店': 6, '箱包店': 6},\n",
       "  '滑县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '淮阳县': {'运动表现店': 1},\n",
       "  '辉县市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '获嘉县': {'运动表现店': 1},\n",
       "  '济源市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '焦作市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10, '箱包店': 10},\n",
       "  '开封市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '兰考县': {'运动表现店': 1},\n",
       "  '林州市': {'运动表现店': 3},\n",
       "  '临颍县': {'运动表现店': 1},\n",
       "  '灵宝市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '鲁山县': {'运动表现店': 1},\n",
       "  '鹿邑县': {'运动表现店': 1},\n",
       "  '洛阳市': {'运动表现店': 30,\n",
       "   '三叶草店': 30,\n",
       "   '运动休闲店': 30,\n",
       "   '童装店': 30,\n",
       "   '箱包店': 30,\n",
       "   '折扣店': 30},\n",
       "  '孟州市': {'运动表现店': 1},\n",
       "  '民权县': {'运动表现店': 1},\n",
       "  '南阳市': {'运动表现店': 14, '运动休闲店': 14, '童装店': 14},\n",
       "  '南召县': {'运动表现店': 1},\n",
       "  '内黄县': {'运动表现店': 1},\n",
       "  '宁陵县': {'运动表现店': 1},\n",
       "  '平顶山市': {'运动表现店': 17, '运动休闲店': 17, '童装店': 17, '箱包店': 17, '折扣店': 17},\n",
       "  '平舆县': {'运动表现店': 2},\n",
       "  '沁阳市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '清丰县': {'运动表现店': 1},\n",
       "  '汝阳县': {'运动表现店': 1},\n",
       "  '汝州市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '三门峡市': {'运动表现店': 6, '运动休闲店': 6},\n",
       "  '商丘市': {'运动表现店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '商水县': {'运动表现店': 1},\n",
       "  '上蔡县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '社旗县': {'运动表现店': 2},\n",
       "  '沈丘县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '太康县': {'运动表现店': 1},\n",
       "  '唐河县': {'运动表现店': 1},\n",
       "  '通许县': {'运动表现店': 1},\n",
       "  '桐柏县': {'运动表现店': 1},\n",
       "  '尉氏县': {'运动表现店': 1},\n",
       "  '卫辉市': {'运动表现店': 1},\n",
       "  '温县': {'运动表现店': 1},\n",
       "  '武陟县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '舞钢市': {'运动表现店': 1},\n",
       "  '西华县': {'运动表现店': 1},\n",
       "  '西平县': {'运动表现店': 1},\n",
       "  '西峡县': {'运动表现店': 2},\n",
       "  '息县': {'运动表现店': 1},\n",
       "  '夏邑县': {'运动表现店': 2},\n",
       "  '襄城县': {'运动表现店': 1},\n",
       "  '项城市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '新安县': {'运动表现店': 1},\n",
       "  '新蔡县': {'运动表现店': 1},\n",
       "  '新密市': {'运动表现店': 7, '运动休闲店': 7},\n",
       "  '新乡市': {'运动表现店': 17, '运动休闲店': 17, '童装店': 17},\n",
       "  '新乡县': {'箱包店': 1},\n",
       "  '新郑市': {'运动表现店': 6, '运动休闲店': 6},\n",
       "  '信阳市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '修武县': {'运动表现店': 1},\n",
       "  '许昌市': {'运动表现店': 4, '三叶草店': 4, '童装店': 4},\n",
       "  '叶县': {'运动表现店': 1},\n",
       "  '伊川县': {'运动表现店': 1},\n",
       "  '宜阳县': {'运动表现店': 1},\n",
       "  '义马市': {'运动表现店': 1},\n",
       "  '永城市': {'运动表现店': 3},\n",
       "  '虞城县': {'运动表现店': 1},\n",
       "  '禹州市': {'运动表现店': 2},\n",
       "  '原阳县': {'运动表现店': 1},\n",
       "  '郑州市': {'迷你品牌中心': 130,\n",
       "   '运动表现店': 130,\n",
       "   '三叶草店': 130,\n",
       "   '运动休闲店': 130,\n",
       "   '篮球店': 130,\n",
       "   '童装店': 130,\n",
       "   '箱包店': 130,\n",
       "   '折扣店': 130},\n",
       "  '中牟县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '周口市': {'运动表现店': 5, '童装店': 5},\n",
       "  '驻马店市': {'运动表现店': 8, '运动休闲店': 8},\n",
       "  '偃师市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '郏县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '鄢陵县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '荥阳市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '淇县': {'运动表现店': 1},\n",
       "  '淅川县': {'运动表现店': 1},\n",
       "  '渑池县': {'运动表现店': 1},\n",
       "  '漯河市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '濮阳市': {'运动表现店': 8, '运动休闲店': 8},\n",
       "  '柘城县': {'运动表现店': 1},\n",
       "  '栾川县': {'运动表现店': 2},\n",
       "  '睢县': {'运动表现店': 1}},\n",
       " '山西': {'长治市': {'运动表现店': 14, '运动休闲店': 14, '童装店': 14},\n",
       "  '大同市': {'运动表现店': 27, '三叶草店': 27, '运动休闲店': 27, '童装店': 27},\n",
       "  '繁峙县': {'运动表现店': 1},\n",
       "  '汾阳市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '高平市': {'运动表现店': 2},\n",
       "  '河津市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '洪洞县': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '侯马市': {'运动表现店': 6, '运动休闲店': 6},\n",
       "  '怀仁县': {'运动表现店': 7, '运动休闲店': 7},\n",
       "  '霍州市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '介休市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '晋城市': {'运动表现店': 20, '运动休闲店': 20, '童装店': 20},\n",
       "  '晋中市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '临汾市': {'运动表现店': 16, '运动休闲店': 16, '童装店': 16},\n",
       "  '吕梁市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '平遥县': {'运动表现店': 1},\n",
       "  '曲沃县': {'运动表现店': 1},\n",
       "  '朔州市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '太谷县': {'运动表现店': 1},\n",
       "  '太原市': {'运动表现店': 75, '三叶草店': 75, '运动休闲店': 75, '童装店': 75, '折扣店': 75},\n",
       "  '文水县': {'运动表现店': 1},\n",
       "  '襄汾县': {'运动表现店': 1},\n",
       "  '孝义市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '新绛县': {'运动表现店': 1},\n",
       "  '忻州市': {'运动表现店': 10, '运动休闲店': 10},\n",
       "  '阳城县': {'运动表现店': 1},\n",
       "  '阳泉市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '翼城县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '永济市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '盂县': {'运动表现店': 1},\n",
       "  '垣曲县': {'运动表现店': 1},\n",
       "  '原平市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '运城市': {'运动表现店': 20, '运动休闲店': 20, '童装店': 20},\n",
       "  '绛县': {'运动表现店': 1},\n",
       "  '稷山县': {'运动表现店': 1}},\n",
       " '北京': {'北京市': {'品牌中心': 404,\n",
       "   '迷你品牌中心': 404,\n",
       "   '运动休闲概念店': 404,\n",
       "   '运动表现店': 404,\n",
       "   '三叶草店': 404,\n",
       "   '运动休闲店': 404,\n",
       "   '鞋店': 404,\n",
       "   '篮球店': 404,\n",
       "   '户外店': 404,\n",
       "   '童装店': 404,\n",
       "   '箱包店': 404,\n",
       "   '折扣店': 404},\n",
       "  '密云县': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8, '折扣店': 8},\n",
       "  '延庆县': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5}},\n",
       " '广东': {'博罗县': {'运动表现店': 2},\n",
       "  '潮安县': {'运动表现店': 1},\n",
       "  '潮州市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '从化市': {'运动表现店': 2},\n",
       "  '电白县': {'运动表现店': 2},\n",
       "  '东莞市': {'运动表现店': 71, '三叶草店': 71, '运动休闲店': 71, '童装店': 71},\n",
       "  '恩平市': {'运动表现店': 2},\n",
       "  '佛冈县': {'运动表现店': 1},\n",
       "  '佛山市': {'运动表现店': 79, '三叶草店': 79, '运动休闲店': 79, '童装店': 79, '折扣店': 79},\n",
       "  '高州市': {'运动表现店': 1},\n",
       "  '广州市': {'品牌中心': 163,\n",
       "   '迷你品牌中心': 163,\n",
       "   '运动休闲概念店': 163,\n",
       "   '运动表现店': 163,\n",
       "   '三叶草店': 163,\n",
       "   '运动休闲店': 163,\n",
       "   '足球店': 163,\n",
       "   '篮球店': 163,\n",
       "   '女子店': 163,\n",
       "   '童装店': 163,\n",
       "   '箱包店': 163,\n",
       "   '折扣店': 163},\n",
       "  '海丰县': {'运动表现店': 1},\n",
       "  '河源市': {'运动表现店': 4},\n",
       "  '鹤山市': {'运动表现店': 2},\n",
       "  '化州市': {'运动表现店': 1},\n",
       "  '怀集县': {'运动表现店': 1},\n",
       "  '惠东县': {'运动表现店': 2},\n",
       "  '惠州市': {'运动表现店': 28, '三叶草店': 28, '运动休闲店': 28, '童装店': 28},\n",
       "  '江门市': {'运动表现店': 14, '三叶草店': 14, '运动休闲店': 14, '童装店': 14, '折扣店': 14},\n",
       "  '揭阳市': {'运动表现店': 2},\n",
       "  '开平市': {'运动表现店': 6},\n",
       "  '雷州市': {'运动表现店': 1},\n",
       "  '廉江市': {'运动表现店': 2},\n",
       "  '龙川县': {'运动表现店': 1},\n",
       "  '罗定市': {'运动表现店': 1},\n",
       "  '茂名市': {'运动表现店': 7, '运动休闲店': 7, '箱包店': 7, '折扣店': 7},\n",
       "  '梅县': {'运动表现店': 1},\n",
       "  '梅州市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '南雄市': {'运动表现店': 1},\n",
       "  '普宁市': {'运动表现店': 4, '童装店': 4},\n",
       "  '清新县': {'运动表现店': 1},\n",
       "  '清远市': {'运动表现店': 8, '童装店': 8},\n",
       "  '汕头市': {'运动表现店': 21, '运动休闲店': 21},\n",
       "  '汕尾市': {'运动表现店': 1},\n",
       "  '韶关市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '深圳市': {'迷你品牌中心': 182,\n",
       "   '运动表现店': 182,\n",
       "   '三叶草店': 182,\n",
       "   '运动休闲店': 182,\n",
       "   '篮球店': 182,\n",
       "   '童装店': 182,\n",
       "   '箱包店': 182,\n",
       "   '折扣店': 182},\n",
       "  '四会市': {'运动表现店': 3},\n",
       "  '遂溪县': {'运动表现店': 1},\n",
       "  '台山市': {'运动表现店': 3},\n",
       "  '吴川市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '新兴县': {'运动表现店': 2},\n",
       "  '信宜市': {'运动表现店': 1},\n",
       "  '兴宁市': {'运动表现店': 1},\n",
       "  '徐闻县': {'运动表现店': 1},\n",
       "  '阳春市': {'运动休闲店': 1},\n",
       "  '阳东县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '阳江市': {'运动表现店': 9, '运动休闲店': 9},\n",
       "  '阳西县': {'运动表现店': 2},\n",
       "  '英德市': {'运动表现店': 2},\n",
       "  '云浮市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '增城市': {'运动表现店': 12, '三叶草店': 12, '运动休闲店': 12},\n",
       "  '湛江市': {'运动表现店': 23, '运动休闲店': 23, '童装店': 23, '箱包店': 23},\n",
       "  '肇庆市': {'运动表现店': 10, '运动休闲店': 10, '童装店': 10},\n",
       "  '中山市': {'运动表现店': 46, '三叶草店': 46, '运动休闲店': 46, '童装店': 46, '箱包店': 46},\n",
       "  '珠海市': {'运动表现店': 30,\n",
       "   '三叶草店': 30,\n",
       "   '运动休闲店': 30,\n",
       "   '童装店': 30,\n",
       "   '箱包店': 30,\n",
       "   '折扣店': 30}},\n",
       " '江苏': {'宝应县': {'运动表现店': 2},\n",
       "  '滨海县': {'运动表现店': 1},\n",
       "  '常熟市': {'运动表现店': 13, '三叶草店': 13, '运动休闲店': 13, '童装店': 13, '箱包店': 13},\n",
       "  '常州市': {'运动表现店': 39, '三叶草店': 39, '运动休闲店': 39, '童装店': 39, '折扣店': 39},\n",
       "  '大丰市': {'运动表现店': 2},\n",
       "  '丹阳市': {'运动表现店': 7, '三叶草店': 7, '运动休闲店': 7, '童装店': 7},\n",
       "  '东海县': {'运动表现店': 2},\n",
       "  '东台市': {'运动表现店': 4},\n",
       "  '阜宁县': {'运动表现店': 1},\n",
       "  '高淳县': {'运动表现店': 4, '童装店': 4},\n",
       "  '高邮市': {'运动表现店': 2},\n",
       "  '海安县': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '海门市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '淮安市': {'运动表现店': 11, '三叶草店': 11, '运动休闲店': 11, '童装店': 11, '折扣店': 11},\n",
       "  '建湖县': {'运动表现店': 1},\n",
       "  '姜堰市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '江都市': {'运动表现店': 2, '童装店': 2},\n",
       "  '江阴市': {'运动表现店': 14, '三叶草店': 14, '运动休闲店': 14, '童装店': 14, '箱包店': 14},\n",
       "  '金坛市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '靖江市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '句容市': {'运动表现店': 2},\n",
       "  '昆山市': {'运动表现店': 25,\n",
       "   '三叶草店': 25,\n",
       "   '运动休闲店': 25,\n",
       "   '童装店': 25,\n",
       "   '箱包店': 25,\n",
       "   '折扣店': 25},\n",
       "  '连云港市': {'运动表现店': 16, '运动休闲店': 16, '童装店': 16},\n",
       "  '南京市': {'运动休闲概念店': 144,\n",
       "   '运动表现店': 144,\n",
       "   '三叶草店': 144,\n",
       "   '运动休闲店': 144,\n",
       "   '篮球店': 144,\n",
       "   '童装店': 144,\n",
       "   '箱包店': 144,\n",
       "   '折扣店': 144},\n",
       "  '南通市': {'迷你品牌中心': 44,\n",
       "   '运动表现店': 44,\n",
       "   '三叶草店': 44,\n",
       "   '运动休闲店': 44,\n",
       "   '童装店': 44,\n",
       "   '折扣店': 44},\n",
       "  '沛县': {'运动表现店': 1},\n",
       "  '启东市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '如东县': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '如皋市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '射阳县': {'运动表现店': 1},\n",
       "  '苏州市': {'运动表现店': 106, '三叶草店': 106, '运动休闲店': 106, '童装店': 106, '折扣店': 106},\n",
       "  '宿迁市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '泰兴市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '泰州市': {'运动表现店': 14, '三叶草店': 14, '运动休闲店': 14, '童装店': 14},\n",
       "  '太仓市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '无锡市': {'迷你品牌中心': 62,\n",
       "   '运动表现店': 62,\n",
       "   '三叶草店': 62,\n",
       "   '运动休闲店': 62,\n",
       "   '篮球店': 62,\n",
       "   '童装店': 62,\n",
       "   '箱包店': 62,\n",
       "   '折扣店': 62},\n",
       "  '响水县': {'运动表现店': 1},\n",
       "  '新沂市': {'运动表现店': 1},\n",
       "  '兴化市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '徐州市': {'运动表现店': 40,\n",
       "   '三叶草店': 40,\n",
       "   '运动休闲店': 40,\n",
       "   '童装店': 40,\n",
       "   '箱包店': 40,\n",
       "   '折扣店': 40},\n",
       "  '盐城市': {'运动表现店': 20, '三叶草店': 20, '运动休闲店': 20, '童装店': 20, '折扣店': 20},\n",
       "  '扬中市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '扬州市': {'运动表现店': 30, '三叶草店': 30, '运动休闲店': 30, '童装店': 30},\n",
       "  '仪征市': {'运动表现店': 2},\n",
       "  '宜兴市': {'运动表现店': 13, '三叶草店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '张家港市': {'运动表现店': 22, '三叶草店': 22, '运动休闲店': 22, '篮球店': 22, '童装店': 22},\n",
       "  '镇江市': {'运动表现店': 19, '三叶草店': 19, '运动休闲店': 19, '童装店': 19, '折扣店': 19},\n",
       "  '邳州市': {'运动表现店': 8, '运动休闲店': 8},\n",
       "  '沭阳县': {'运动表现店': 2, '童装店': 2},\n",
       "  '泗洪县': {'运动表现店': 2},\n",
       "  '泗阳县': {'运动表现店': 3, '童装店': 3},\n",
       "  '溧水县': {'运动表现店': 1},\n",
       "  '溧阳市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '盱眙县': {'运动表现店': 3},\n",
       "  '睢宁县': {'运动表现店': 4, '运动休闲店': 4}},\n",
       " '天津': {'蓟县': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '静海县': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '天津市': {'品牌中心': 136,\n",
       "   '运动表现店': 136,\n",
       "   '三叶草店': 136,\n",
       "   '运动休闲店': 136,\n",
       "   '篮球店': 136,\n",
       "   '童装店': 136,\n",
       "   '折扣店': 136}},\n",
       " '湖南': {'安乡县': {'运动休闲店': 1},\n",
       "  '常德市': {'运动表现店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '长沙市': {'迷你品牌中心': 116,\n",
       "   '运动表现店': 116,\n",
       "   '三叶草店': 116,\n",
       "   '运动休闲店': 116,\n",
       "   '篮球店': 116,\n",
       "   '童装店': 116,\n",
       "   '箱包店': 116,\n",
       "   '折扣店': 116},\n",
       "  '长沙县': {'运动表现店': 4, '三叶草店': 4, '童装店': 4},\n",
       "  '郴州市': {'运动表现店': 14, '运动休闲店': 14, '童装店': 14, '箱包店': 14},\n",
       "  '衡阳市': {'运动表现店': 19, '运动休闲店': 19, '童装店': 19},\n",
       "  '花垣县': {'运动表现店': 1},\n",
       "  '怀化市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '吉首市': {'运动表现店': 6, '运动休闲店': 6, '童装店': 6},\n",
       "  '涟源市': {'运动表现店': 2},\n",
       "  '龙山县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '娄底市': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '宁乡县': {'运动表现店': 4, '运动休闲店': 4, '童装店': 4},\n",
       "  '邵阳市': {'运动表现店': 1},\n",
       "  '武冈市': {'运动表现店': 1},\n",
       "  '湘潭市': {'运动表现店': 10, '三叶草店': 10, '运动休闲店': 10, '童装店': 10, '箱包店': 10},\n",
       "  '湘潭县': {'运动表现店': 1},\n",
       "  '湘乡市': {'运动表现店': 1},\n",
       "  '新化县': {'运动休闲店': 1},\n",
       "  '星沙县': {'运动表现店': 1},\n",
       "  '益阳市': {'运动表现店': 4, '运动休闲店': 4},\n",
       "  '永兴县': {'运动表现店': 1},\n",
       "  '永州市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '岳阳市': {'运动表现店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '张家界市': {'运动表现店': 3, '童装店': 3},\n",
       "  '株洲市': {'运动表现店': 14, '运动休闲店': 14, '童装店': 14, '箱包店': 14},\n",
       "  '攸县': {'运动表现店': 1},\n",
       "  '沅江市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '汨罗市': {'运动表现店': 1},\n",
       "  '浏阳市': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '溆浦县': {'运动表现店': 1},\n",
       "  '澧县': {'运动表现店': 2, '童装店': 2},\n",
       "  '耒阳市': {'运动表现店': 2, '运动休闲店': 2}},\n",
       " '陕西': {'安康市': {'运动表现店': 8, '运动休闲店': 8, '童装店': 8},\n",
       "  '宝鸡市': {'运动表现店': 13, '三叶草店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '澄城县': {'运动表现店': 1},\n",
       "  '大荔县': {'运动表现店': 1},\n",
       "  '定边县': {'运动表现店': 2, '童装店': 2},\n",
       "  '府谷县': {'运动表现店': 1},\n",
       "  '富县': {'运动表现店': 2},\n",
       "  '韩城市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '汉中市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '华县': {'运动表现店': 1},\n",
       "  '华阴市': {'运动表现店': 1},\n",
       "  '靖边县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '蓝田县': {'运动表现店': 1},\n",
       "  '洛南县': {'运动表现店': 1},\n",
       "  '勉县': {'运动表现店': 1},\n",
       "  '蒲城县': {'运动表现店': 1},\n",
       "  '商洛市': {'运动表现店': 2},\n",
       "  '神木县': {'运动表现店': 2, '童装店': 2},\n",
       "  '绥德县': {'运动休闲店': 1},\n",
       "  '铜川市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '渭南市': {'运动表现店': 11, '运动休闲店': 11, '童装店': 11},\n",
       "  '吴起县': {'运动表现店': 2},\n",
       "  '西安市': {'迷你品牌中心': 194,\n",
       "   '运动休闲概念店': 194,\n",
       "   '运动表现店': 194,\n",
       "   '三叶草店': 194,\n",
       "   '运动休闲店': 194,\n",
       "   '篮球店': 194,\n",
       "   '童装店': 194,\n",
       "   '折扣店': 194},\n",
       "  '咸阳市': {'运动表现店': 9, '运动休闲店': 9, '童装店': 9},\n",
       "  '兴平市': {'运动表现店': 1},\n",
       "  '延安市': {'运动表现店': 13, '运动休闲店': 13, '童装店': 13},\n",
       "  '榆林市': {'运动表现店': 16, '运动休闲店': 16, '童装店': 16},\n",
       "  '镇安县': {'运动表现店': 1},\n",
       "  '志丹县': {'运动表现店': 1},\n",
       "  '周至县': {'运动表现店': 1},\n",
       "  '子长县': {'运动表现店': 1}},\n",
       " '福建': {'安溪县': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '长乐市': {'运动表现店': 1},\n",
       "  '长汀县': {'运动表现店': 1},\n",
       "  '德化县': {'运动表现店': 1},\n",
       "  '福安市': {'运动表现店': 1},\n",
       "  '福鼎市': {'运动表现店': 2},\n",
       "  '福清市': {'运动表现店': 6, '运动休闲店': 6},\n",
       "  '福州市': {'迷你品牌中心': 71,\n",
       "   '运动表现店': 71,\n",
       "   '三叶草店': 71,\n",
       "   '运动休闲店': 71,\n",
       "   '篮球店': 71,\n",
       "   '童装店': 71,\n",
       "   '箱包店': 71,\n",
       "   '折扣店': 71},\n",
       "  '古田县': {'运动表现店': 1},\n",
       "  '惠安县': {'运动表现店': 8, '运动休闲店': 8},\n",
       "  '建阳市': {'运动表现店': 1},\n",
       "  '晋江市': {'运动表现店': 12, '三叶草店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '连江县': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '龙海市': {'运动表现店': 3, '运动休闲店': 3},\n",
       "  '龙岩市': {'运动表现店': 12, '三叶草店': 12, '运动休闲店': 12, '童装店': 12},\n",
       "  '闽侯县': {'运动表现店': 5, '运动休闲店': 5, '童装店': 5},\n",
       "  '南安市': {'运动表现店': 5},\n",
       "  '南平市': {'运动表现店': 1},\n",
       "  '宁德市': {'三叶草店': 1},\n",
       "  '宁化县': {'运动表现店': 1},\n",
       "  '平和县': {'运动表现店': 1},\n",
       "  '莆田市': {'运动表现店': 6, '运动休闲店': 6},\n",
       "  '泉州市': {'运动表现店': 24, '运动休闲店': 24, '童装店': 24},\n",
       "  '三明市': {'运动表现店': 5, '运动休闲店': 5},\n",
       "  '沙县': {'运动表现店': 1},\n",
       "  '石狮市': {'运动表现店': 8, '运动休闲店': 8},\n",
       "  '武夷山市': {'运动表现店': 1},\n",
       "  '霞浦县': {'运动表现店': 1},\n",
       "  '厦门市': {'迷你品牌中心': 88,\n",
       "   '运动表现店': 88,\n",
       "   '三叶草店': 88,\n",
       "   '运动休闲店': 88,\n",
       "   '童装店': 88,\n",
       "   '箱包店': 88,\n",
       "   '折扣店': 88},\n",
       "  '仙游县': {'运动表现店': 1},\n",
       "  '永安市': {'运动表现店': 2, '运动休闲店': 2},\n",
       "  '云霄县': {'运动表现店': 1},\n",
       "  '漳浦县': {'运动表现店': 1},\n",
       "  '漳州市': {'运动表现店': 14, '三叶草店': 14, '运动休闲店': 14}}}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 数据整理与输出"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('adidas_txt00.txt','w') as f:\n",
    "    f.write(str(res))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "31"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(res)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "len(province)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "cck=0\n",
    "for ki,vi in res.items():\n",
    "    for kj,vj in vi.items():\n",
    "        for k,v in vj.items():\n",
    "            cck+=v\n",
    "            "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "10096"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "cck #一共多少家店"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2572"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sww=[]\n",
    "for ki,vi in res.items():\n",
    "    for kj,vj in vi.items():\n",
    "        for k,v in vj.items():\n",
    "            sww.append({'province':ki,'city':kj,'stype':k,'stcount':v})\n",
    "len(sww)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cck"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>city</th>\n",
       "      <th>province</th>\n",
       "      <th>stcount</th>\n",
       "      <th>stype</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>安国市</td>\n",
       "      <td>河北</td>\n",
       "      <td>1</td>\n",
       "      <td>运动表现店</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>安平县</td>\n",
       "      <td>河北</td>\n",
       "      <td>1</td>\n",
       "      <td>运动表现店</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  city province  stcount  stype\n",
       "0  安国市       河北        1  运动表现店\n",
       "1  安平县       河北        1  运动表现店"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sdf=pd.DataFrame(sww)\n",
    "sdf.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "sdf.to_excel('./adidas/adidas-表现店统计.xlsx')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "kdf=pd.DataFrame(klst)\n",
    "kdf.to_excel('./adidas/adidas-表现店详情.xlsx')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>addressCN</th>\n",
       "      <th>addressEN</th>\n",
       "      <th>city</th>\n",
       "      <th>id</th>\n",
       "      <th>latitude</th>\n",
       "      <th>longitude</th>\n",
       "      <th>pOSNameCN</th>\n",
       "      <th>pOSNameEN</th>\n",
       "      <th>province</th>\n",
       "      <th>storeType</th>\n",
       "      <th>storeTypeEN</th>\n",
       "      <th>telephone</th>\n",
       "      <th>workHourCN</th>\n",
       "      <th>workHourEn</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>河北安国市药华大街170号宜家旺商场4楼</td>\n",
       "      <td>河北安国市药华大街170号宜家旺商场4楼</td>\n",
       "      <td>安国市</td>\n",
       "      <td>5351</td>\n",
       "      <td>38.422413</td>\n",
       "      <td>115.321785</td>\n",
       "      <td>安国市药华大街宜家旺商场运动表现店</td>\n",
       "      <td>安国市药华大街宜家旺商场运动表现店</td>\n",
       "      <td>河北</td>\n",
       "      <td>运动表现店</td>\n",
       "      <td>Sport Performance</td>\n",
       "      <td>18832238828</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>河北安平县为民街22号瑞泰广场2楼</td>\n",
       "      <td>河北安平县为民街22号瑞泰广场2楼</td>\n",
       "      <td>安平县</td>\n",
       "      <td>6203</td>\n",
       "      <td>38.233459</td>\n",
       "      <td>115.52456</td>\n",
       "      <td>安平县为民街瑞泰广场运动表现店</td>\n",
       "      <td>安平县为民街瑞泰广场运动表现店</td>\n",
       "      <td>河北</td>\n",
       "      <td>运动表现店</td>\n",
       "      <td>Sport Performance</td>\n",
       "      <td>15076826658</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>河北安新县新南大街31号华傲购物中心1楼</td>\n",
       "      <td>河北安新县新南大街31号华傲购物中心1楼</td>\n",
       "      <td>安新县</td>\n",
       "      <td>4407</td>\n",
       "      <td>38.9183262</td>\n",
       "      <td>115.93166742</td>\n",
       "      <td>安新县新南大街31号运动表现店</td>\n",
       "      <td>安新县新南大街31号运动表现店</td>\n",
       "      <td>河北</td>\n",
       "      <td>运动表现店</td>\n",
       "      <td>Sport Performance</td>\n",
       "      <td>13833092206</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "              addressCN             addressEN city    id    latitude  \\\n",
       "0  河北安国市药华大街170号宜家旺商场4楼  河北安国市药华大街170号宜家旺商场4楼  安国市  5351   38.422413   \n",
       "1     河北安平县为民街22号瑞泰广场2楼     河北安平县为民街22号瑞泰广场2楼  安平县  6203   38.233459   \n",
       "2  河北安新县新南大街31号华傲购物中心1楼  河北安新县新南大街31号华傲购物中心1楼  安新县  4407  38.9183262   \n",
       "\n",
       "      longitude          pOSNameCN          pOSNameEN province storeType  \\\n",
       "0    115.321785  安国市药华大街宜家旺商场运动表现店  安国市药华大街宜家旺商场运动表现店       河北     运动表现店   \n",
       "1     115.52456    安平县为民街瑞泰广场运动表现店    安平县为民街瑞泰广场运动表现店       河北     运动表现店   \n",
       "2  115.93166742    安新县新南大街31号运动表现店    安新县新南大街31号运动表现店       河北     运动表现店   \n",
       "\n",
       "         storeTypeEN    telephone workHourCN workHourEn  \n",
       "0  Sport Performance  18832238828                        \n",
       "1  Sport Performance  15076826658                        \n",
       "2  Sport Performance  13833092206                        "
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "kdf.head(3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "10101"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(kdf)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "河北 西藏 浙江 甘肃 江西 辽宁 重庆 山东 海南 云南 贵州 安徽 吉林 青海 四川 内蒙古 湖北 宁夏 广西 新疆 上海 黑龙江 河南 山西 北京 广东 江苏 天津 湖南 陕西 福建 "
     ]
    }
   ],
   "source": [
    "for ki,vi in res.items():\n",
    "    df=pd.DataFrame(vi)\n",
    "    df.to_excel('./adidas/adidas-{0}.xlsx'.format(ki),sheet_name=str(ki))\n",
    "    print(ki,end=' ')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "### 测试用"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import yaml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('adidas_txt01.txt','r') as f:\n",
    "    s=f.readlines()[0]\n",
    "    js=yaml.load(s)\n",
    "    print(js)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ktt=[]\n",
    "cc=0\n",
    "for pkey,pval in province.items():\n",
    "    prtt=['','','','']\n",
    "    try:\n",
    "        cc+=1\n",
    "        pv=pval.replace('省','')\n",
    "        prtt[0]=pv\n",
    "        print(pv,cc,end=' ')\n",
    "        res[pv]={}\n",
    "        purl='https://www.adidas.com.cn/location/city.json?province={0}&_=1556377173555'.format(pv)\n",
    "        cts=getOneUrl(purl)\n",
    "        if cts==None:\n",
    "            print('cts==None',pv,purl)\n",
    "            cts='[]'\n",
    "        citys=json.loads(cts)\n",
    "        for c in citys:\n",
    "            curl='https://www.adidas.com.cn/location/storelist.json?startPage=1&size=12&province={0}&city={1}&storeName=&_=1556376900755'.format(pv,c)\n",
    "            crtxt=getOneUrl(curl)\n",
    "            cjs=json.loads(crtxt)\n",
    "            ktt.extend(cjs['storeMapResult'])\n",
    "            cc+=1\n",
    "            if cc%5==0:\n",
    "                time.sleep(1)\n",
    "            \n",
    "    except:\n",
    "        traceback.print_exc()\n",
    "        print(prtt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "print(len(ktt))\n",
    "ktt[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "prov={1:'辽宁',2:'广东'}\n",
    "cc=0\n",
    "for pkey,pval in prov.items():\n",
    "    prtt=['','','','']\n",
    "    try:\n",
    "        cc+=1\n",
    "        pv=pval.replace('省','')\n",
    "        prtt[0]=pv\n",
    "        print(pv,cc,end=' ')\n",
    "        res[pv]={}\n",
    "        purl='https://www.adidas.com.cn/location/city.json?province={0}&_=1556377173555'.format(pv)\n",
    "        cts=getOneUrl(purl)\n",
    "        if cts==None:\n",
    "            print('cts==None',pv,purl)\n",
    "            cts='[]'\n",
    "        citys=json.loads(cts)\n",
    "        for c in citys:\n",
    "            res[pv][c]={}\n",
    "            prtt[1]=c\n",
    "            sts=getStoreTypes(c) #sotretypes list str\n",
    "            for s in sts:\n",
    "                prtt[2]=s\n",
    "                stotal=getSTotal(pv,c,s)\n",
    "                res[pv][c][s]=stotal[0]\n",
    "                klst.extend(stotal[1])\n",
    "                prtt[3]=stotal\n",
    "                #time.sleep(1)\n",
    "            time.sleep(1)\n",
    "            \n",
    "    except:\n",
    "        traceback.print_exc()\n",
    "        print(prtt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "prov={1:'辽宁'}\n",
    "cc=0\n",
    "for pv in ['辽宁']:\n",
    "    prtt=['','','','']\n",
    "    try:\n",
    "        prtt[0]=pv\n",
    "        print(pv,cc,end=' ')\n",
    "        res[pv]={}\n",
    "        for c in ['新民市']:\n",
    "            res[pv][c]={}\n",
    "            prtt[1]=c\n",
    "            sts=getStoreTypes(c) #sotretypes list str\n",
    "            for s in sts:\n",
    "                prtt[2]=s\n",
    "                stotal=getSTotal(pv,c,s)\n",
    "                res[pv][c][s]=stotal[0]\n",
    "                klst.extend(stotal[1])\n",
    "                prtt[3]=stotal[0]\n",
    "                #time.sleep(1)\n",
    "            time.sleep(1)\n",
    "            \n",
    "    except:\n",
    "        traceback.print_exc()\n",
    "        print(prtt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "ress"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#{'省':{'市':{'店':个数}}}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#要坐标直接写 s['storeMapResult']   not  s['storeResult']  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cts=requests.get('https://www.adidas.com.cn/location/city.json?province={0}&_=1556377173555'.format('江西')).text\n",
    "citys=json.loads(cts)\n",
    "citys"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "sotretypes[0]['storeType']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'storeResult': [{'addressCN': '江西东乡县财富广场号东信百货1楼',\n",
       "   'addressEN': '江西东乡县财富广场号东信百货1楼',\n",
       "   'city': '东乡县',\n",
       "   'id': 4955,\n",
       "   'latitude': '28.242789',\n",
       "   'longitude': '116.608582',\n",
       "   'pOSNameCN': '东乡县财富广场东信百货运动表现店',\n",
       "   'pOSNameEN': '东乡县财富广场东信百货运动表现店',\n",
       "   'province': '江西',\n",
       "   'storeType': '运动表现店',\n",
       "   'storeTypeEN': 'Sport Performance',\n",
       "   'telephone': '0794-4236288',\n",
       "   'workHourCN': '',\n",
       "   'workHourEn': ''}],\n",
       " 'storeMapResult': [{'addressCN': '江西东乡县财富广场号东信百货1楼',\n",
       "   'addressEN': '江西东乡县财富广场号东信百货1楼',\n",
       "   'city': '东乡县',\n",
       "   'id': 4955,\n",
       "   'latitude': '28.242789',\n",
       "   'longitude': '116.608582',\n",
       "   'pOSNameCN': '东乡县财富广场东信百货运动表现店',\n",
       "   'pOSNameEN': '东乡县财富广场东信百货运动表现店',\n",
       "   'province': '江西',\n",
       "   'storeType': '运动表现店',\n",
       "   'storeTypeEN': 'Sport Performance',\n",
       "   'telephone': '0794-4236288',\n",
       "   'workHourCN': '',\n",
       "   'workHourEn': ''}],\n",
       " 'pageTotal': 1,\n",
       " 'startPage': '1',\n",
       " 'storeTotal': 1}"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "stt=json.loads(requests.get(\n",
    "                'https://www.adidas.com.cn/location/storelist.json?startPage=1&size=12&province={0}&city={1}&storeName=&types={2}&_=1556376900755'.format('江西','东乡县','运动表现店')).text)\n",
    "stt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
